### PR TITLE
[MRG] Rename feeds to drives

### PR DESCRIPTION
**Pull request by jasmainak**
Talking to @stephanie-r-jones I realized that it's confusing for new developers or someone reading the code to see both `feed.py` and `drives.py`. Trying to convert everything to "drive" nomenclature here

### CODE REVIEW

**Comment by jasmainak** [comment]
any objections @cjayb ?

**Comment by cjayb** [comment]
I left `feed.py` and the variable names in there until we could deprecate it. I think we should keep our current tests until GUI integration is complete so we can verify that it can reproduce the examples. But I'm OK with going ahead with `feed`->`drive` now.

**Comment by jasmainak** [comment]
I see. This PR does nothing more than renaming. Public API is not touched.

Okay I'm done here. I'm happy if CIs are happy

File: hnn_core/tests/test_drives.py [hunk 1]
```diff
@@ -2,11 +2,113 @@
...
+                             _create_bursty_input)
+from hnn_core.params import create_pext
+
+
+def test_extfeed():
+    """Test the different external feeds."""
+
+    params = Params()
+    p_common, p_unique = create_pext(params,
+                                     params['tstop'])
```
[the remark this answers is not in the document]
Comment (cjayb) [reply hnn_core/tests/test_drives.py]: `test_drives()` currently does argument checking _using the new API_. This is reading `params` and testing that the spike times match expected values.

**Review by cjayb — APPROVED** [review]

**Comment by cjayb** [comment]
> I'm happy if CIs are happy

aren't we all! Looks like you're hitting the wall that is: "we must be able to reproduce HNN GUI output".

ALL tests are using the old GUI-derived names ('extpois', 'evdist2', 'common', etc.) so when you change everything to use `drive_event_times`, things break down. The tests require that those specific names (all of them) exist.

This isn't pretty. Leaving `feed_event_times` in there to pass tests was admittedly an Ugly Hack from my part.

**Comment by jasmainak** [comment]
it was an issue with the rebase. Silly me!

**Comment by cjayb** [comment]
Thanks for the cleanup @jasmainak ! I didn't see the rebase issue either :)

**Pull request merged**